js/Features.js

```diff
@@ -152,13 +152,13 @@ Features.prototype.updateData = function (featureElement) {
             var group = featureElement.attr('name');
             var controlElements = $('input[name="' + group + '"]');
             var selectedBit = controlElements.filter(':checked').val();
-
+            var thisFeature = this;
```

Comment:
I think it would be better to only add `_featureMask` to the closure, instead of a back reference to the whole object, as that's the only property that's accessed. That would also bring this method in line with how it's done in the other methods: 

https://github.com/KiteAnton/betaflight-configurator/blob/12636365fc5fc1b97a592d9630e4eb735992fe85/js/Features.js#L72